client/src/client-only-routes/ShowProjectLinks.js

```diff
@@ -142,7 +150,12 @@ const ShowProjectLinks = props => {
       ({ link, title, id }) => (
         <li key={id}>
           <Link to={link} className='project-link'>
-            {title}
+            <Trans
+              i18nKey={`certification.project.title.${title}`}
+              defaults={title}
+            >
+              placeholder
+            </Trans>
```

Comment:
```suggestion
            {t(`certification.project.title.${title}`,title)}
```
Same principle as above.